src/App.tsx

```diff
@@ -12,154 +17,21 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          <NavLink to="/" className={getTabClass}>
             Home
-          </a>
+          </NavLink>
 
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
+          <NavLink to="/people" className={getTabClass}>
```

Comment:
Similar to the previous comment, ensure that `getTabClass` is used as a function: `className={({ isActive }) => getTabClass({ isActive })}`.